src/status_im/subs/profile.cljs

```diff
@@ -26,10 +25,20 @@
 
 (re-frame/reg-sub
  :profile/currency-symbol
+ :<- [:currencies]
  :<- [:profile/currency]
- (fn [currency-id]
-   (-> (get currency/currencies currency-id)
-       :symbol)))
+ (fn [[currencies currency-id]]
+   (let [currency (get currencies currency-id)]
+     (if (:token? currency)
+       (:code currency)
+       (:symbol currency)))))
+
+(re-frame/reg-sub
+ :profile/currency-info
+ :<- [:currencies]
+ :<- [:profile/currency]
+ (fn [[currencies currency-id]]
+   (get currencies currency-id)))
```

Comment:
Should the sub be called `:profile/currency-id` instead?

Reply:
it return whole currency object, not only id

Reply:
Makes sense! :+1:  thank you for the clarification!